### ISSUE TITLE
convertToTriangles documentation is confusing

### ISSUE THREAD

**Issue by gregtatum** [Erkaman/gl-catmull-clark#1]
For what it's worth, I found it very confusing that convertToTriangles was true by default. I spent a good half an hour debugging my code before I realized that. I think a less confusing function signature would be `function catmullClark(positions, cells, numSubdivisions[, keepAsQuads])`. I assumed incorrectly that undefined would be falsey.

**Comment by Erkaman** [comment]
Yeah, sorry about that. I will try to look at this in the weekend.

**Comment by gregtatum** [comment]
No worries, it worked like a charm once I figured it out! Thanks for publishing this module, it's really handy: http://sessions.gregtatum.com/013/
